wesnoth-wof.po

```diff
@@ -103,7 +103,7 @@ msgstr "Alfa testování"
 #. [about]
 #: data/campaigns/Winds_of_Fate/about.cfg:82
 msgid "Based on Wings of Victory by:"
-msgstr ""
+msgstr "Námět"
```

Comment:
Já bych spíš přidal, než vypouštěl. Pokud se ti můj návrh nelíbí, vrátil bych tam alespoň tu dvojtečku.
```suggestion
msgstr "Podle tažení Křídla vítězství od:"
```

Reply:
No, já jsem nevěděl, jestli tím námětem je tažení nebo povídka či co, tak jsem se do toho nechtěl pouštět.

Reply:
Udělal jsem to jinak. Hlavně se mi nelíbilo, že by tam figurovalo _od_ ve spojení s následným jménem v prvním pádě.